matrix/MyMatrix.py

```diff
@@ -1,22 +1,114 @@
 class MyMatrix:
-    matrix = [[]]
-
-    def __init__(self, matrix):
+    def __init__(self, matrix, vector, results):
         self.matrix = matrix
+        self.size = len(matrix[0])
+        self.vector = vector
+        self.resultsGeneratedIncreateMatrix = results
+        self.results = [0 for i in range(0, self.size)]
+
+    # def __str__(self):
+    #     string = ""
+    #     for i in (0, self.size):
+    #        for j in (0, self.size):
+    #            string += (self.matrix[i][j])
+    #     return string
+    # to tak średnio chce działać, ale to działa jak toString() z javy
```

Comment:
Czemu tak średnio? Nic się nie pokazuje, czy jak gdy zrobisz `print(matrix)`?

Reply:
Drukuje adres obiektu.

Reply:
Hm, to może coś jest nie tak z przeciążeniem tego
To na razie zostaw może, ja spróbuję to ogarnąć dziś wieczorem